src/components/Footer.tsx

```diff
@@ -78,8 +61,15 @@ export const Footer = () => {
         justify={{ base: "center", md: "space-between" }}
         align={{ base: "center", md: "center" }}
       >
-        <Logo />
-        <Text>Komisja ds. Informatyzacji</Text>
+        <Image src={logoPwr as string} alt="logo-pwr" object-fit="contain" />
+        <Image
+          src={logoSSPwr as string}
+          alt="logo-sspwr"
+          object-fit="contain"
```

Comment:
is there need for this parameter?
Right now every logo is in original size, in the next task there will be need to scale them.